Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -490,7 +487,15 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     else if (args_tab_view == "network")
         tabwidget.set_active_widget(tabwidget.find_descendant_of_type_named<GUI::Widget>("network"));
 
-    return app->exec();
+    int exec = app->exec();
+
+    // When exiting the application, save the configuration of the columns
+    // to be loaded the next time the application is opened.
+    auto& process_table_header = process_table_view.column_header();
+    for (auto column = 0; column < ProcessModel::Column::__Count; ++column)
+        Config::write_bool("SystemMonitor"sv, "ProcessesPreferences"sv, ProcessModel::the().column_name(column), process_table_header.is_section_visible(column));
```

Comment:
"ProcessesPreferences" doesn't make it clear that this is for the table columns. I think "ProcessTableColumns" would be better. (This needs to be changed on line 299 too of course.)